src/package.xml

```diff
@@ -26,5 +26,5 @@
         <members>Inventory Managers</members>
         <name>Profile</name>
     </types>
-    <version>30.0</version>
+    <version>40.0</version>
```

Comment:
This component uses an obsolete API version. Please upgrade to Winter '18 (41.0) or later.